src/components/ConfigSelect/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+
+import { 
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+  Picker
+} from '../Globals/ConfigMetric';
+
+import ConfigSelectProps from 'src/interfaces/ConfigSelectProps';
```

Comment:
I think you can just declare this interface here rather than in `interfaces` because these will never be used outside of this